modules/auxiliary/scanner/http/owa_login.rb

```diff
@@ -79,7 +79,7 @@ def initialize
     register_options(
       [
         OptInt.new('RPORT', [ true, "The target port", 443]),
-        OptAddress.new('RHOST', [ true, "The target address", true]),
+        OptAddress.new('RHOST', [ true, "The target address", '']),
```

Comment:
Just a general once-through to clean it up. It's been worked on by so many people, some of which are our pentesters.